presidio-anonymizer/presidio_anonymizer/anonymizers/validators.py

```diff
@@ -2,8 +2,27 @@
 from presidio_anonymizer.entities import InvalidParamException
 
 
+def validate_parameter_in_range(
+    values_range, parameter_value, parameter_name: str, parameter_type: type
```

Comment:
do we need the type if we use range?

Reply:
done

Reply:
I meant maybe not pass parameter_type at all?